<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>iA4DCTDefectView</class>
 <widget class="QDockWidget" name="iA4DCTDefectView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QRadioButton" name="rbXY">
         <property name="text">
          <string>XY</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbXZ">
         <property name="text">
          <string>XZ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbYZ">
         <property name="text">
          <string>YZ</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QCheckBox" name="cbPullouts">
         <property name="text">
          <string>Pull-outs</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbCracks">
         <property name="text">
          <string>Cracks</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbBreakages">
         <property name="text">
          <string>Breakages</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbDebondings">
         <property name="text">
          <string>Debondings</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="slicerWidget" native="true"/>
      </item>
      <item>
       <widget class="QScrollBar" name="verticalScrollBar">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
